docs/installingLocal.md

```diff
@@ -13,10 +13,16 @@ This page describes the procedure to run PrairieLearn within Docker, but using a
 - Run PrairieLearn with:
 
   ```sh
-  docker run -it --rm -p 3000:3000 -w /PrairieLearn -v /path/to/PrairieLearn:/PrairieLearn prairielearn/prairielearn /bin/bash
+  docker run -it --rm -p 3000:3000 -w /PrairieLearn -v .:/PrairieLearn prairielearn/prairielearn /bin/bash
+  ```
+
+  This will launch a shell inside a Docker container running the PrairieLearn image, but using the current working directory for its code. If you'd rather run the command from somewhere other than the root of the repo, replace `.` with the path to the directory in `.:/PrairieLearn`.
 
-  # You can now run the following commands inside the container:
+  If you get an error like `no matching manifest for linux/arm64/v8 in the manifest list entries` (if you're running for example on a recent Mac), then add `--platform linux/x86_64`.
```

Comment:
Erring on the side of being explicit, since this won't work if you just tack it on after `/bin/bash`:

```suggestion
  If you're running on an Apple Silicon Mac or another ARM-based machine, you may get an error like `no matching manifest for linux/arm64/v8 in the manifest list entries`. To resolve this, add `--platform linux/x86_64` before the image in the command (`prairielearn/prairielearn`).
```